sat/apiclient/fas.py

```diff
@@ -324,8 +339,7 @@ def _create_temporary_snapshot(self, xnames=None):
             now.year, now.month, now.day, now.hour, now.minute, now.second
         )
 
-        # create a system snapshot set to expire in 10 minutes
-        payload = {'name': name, 'expirationTime': later.isoformat() + 'Z'}
+        payload = {'name': name}
```

Comment:
the `later` variable is no longer used, which might be a little confusing.

One idea might be to rename this method  to `_create_snapshot` and have it take a boolean argument `expires` or `temporary` that controls whether the snapshot expires. Another idea could be to just rename it to `_create_snapshot` and get rid of the expiration logic and the `later` variable entirely.

Reply:
I prefer to remove the expiration logic completely as it is nowhere used and renaming the function to `create_snapshot`